attestation-service/Cargo.toml

```
@@ -65,6 +65,7 @@ tokio.workspace = true
 tonic = { workspace = true, optional = true }
 uuid = { version = "1.1.2", features = ["v4"] }
 verifier = { path = "../deps/verifier", default-features = false }
+actix-cors = "0.7"
```

Let's reorder it with alphabet order. And we can use
```suggestion
actix-cors = { version = "0.7", optional = true }
```
and enable this dependency only when we are building restful-as (the front)
```toml
[[bin]]
name = "restful-as"
required-features = ["restful-bin", "actix-cors"]
```